enos/modules/aws_client_windows/main.tf

```diff
@@ -0,0 +1,347 @@
+  test_dir              = "C:/Test/" # needs to end in a / to ensure it creates the directory
+}
+
+resource "local_sensitive_file" "private_key" {
+  depends_on = [tls_private_key.rsa-4096-key]
+
+  content         = tls_private_key.rsa-4096-key.private_key_pem
+  filename        = "${path.root}/.terraform/tmp/key-client-${timestamp()}"
+  file_permission = "0400"
+}
+
+# wait for the SSH service to be available on the instance. We specifically use
+# BatchMode=Yes to prevent SSH from prompting for a password to ensure that we
+# can just SSH using the private key
+resource "enos_local_exec" "wait_for_ssh" {
+  depends_on = [aws_instance.client]
+  inline     = ["timeout 600s bash -c 'until ssh -i ${abspath(local_sensitive_file.private_key.filename)} -o BatchMode=Yes -o IdentitiesOnly=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no Administrator@${aws_instance.client.public_ip} \"echo ready\"; do sleep 10; done'"]
+}
+
+resource "enos_local_exec" "make_dir" {
+  count = var.boundary_cli_zip_path != "" ? 1 : 0
+  depends_on = [
+    enos_local_exec.wait_for_ssh,
+  ]
+
+  inline = ["ssh -i ${abspath(local_sensitive_file.private_key.filename)} -o IdentitiesOnly=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no Administrator@${aws_instance.client.public_ip} mkdir -Force ${local.test_dir}"]
+}
+
+# copy the boundary cli zip file onto the windows client
+resource "enos_local_exec" "add_boundary_cli" {
+  count = var.boundary_cli_zip_path != "" ? 1 : 0
+  depends_on = [
+    local_sensitive_file.private_key,
+    enos_local_exec.make_dir,
+  ]
+
+  inline = ["scp -i ${abspath(local_sensitive_file.private_key.filename)} -o IdentitiesOnly=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ${local.boundary_cli_zip_path} Administrator@${aws_instance.client.public_ip}:${local.test_dir}"]
+}
+
+resource "archive_file" "boundary_ui_src_zip" {
+  count       = var.boundary_ui_src_path != "" ? 1 : 0
+  type        = "zip"
+  source_dir  = var.boundary_ui_src_path
+  output_path = "${path.root}/.terraform/tmp/boundary-ui-src.zip"
+  excludes    = ["**/node_modules/**", "bin/**", "**/out/**"]
+}
+
+resource "enos_local_exec" "add_boundary_ui_src" {
+  count = var.boundary_ui_src_path != "" ? 1 : 0
+  depends_on = [
+    enos_local_exec.make_dir,
+    archive_file.boundary_ui_src_zip
+  ]
+
+  inline = ["scp -i ${abspath(local_sensitive_file.private_key.filename)} -o IdentitiesOnly=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ${archive_file.boundary_ui_src_zip[0].output_path} Administrator@${aws_instance.client.public_ip}:${local.test_dir}"]
+}
+
+resource "archive_file" "boundary_src_zip" {
+  count       = var.boundary_src_path != "" ? 1 : 0
+  type        = "zip"
+  source_dir  = var.boundary_src_path
+  output_path = "${path.root}/.terraform/tmp/boundary-src.zip"
+  excludes    = ["**/enos/**", "**/node_modules/**", "bin/**", "**/.git/**", "plugins/**/*.gz"]
```

Comment:
We could also add `website/**` to this list. When I was testing locally, there were some large files here since I've built the website locally before.